spec/models/remote_rental_spec.rb

```diff
@@ -11,6 +11,28 @@
   it { is_expected.to validate_uniqueness_of(:uid).allow_nil }
   it { is_expected.to validate_presence_of(:remote_account) }
 
+  describe '.connected' do
+    subject { described_class.connected }
+    let(:connection_1) { create(:connection) }
+    let(:remote_rental_1) { connection_1.remote_rental }
+    let(:connection_2) { create(:canceled_connection) }
+    let(:remote_rental_2) { connection_2.remote_rental }
+    let(:remote_rental_3) { create(:remote_rental, rental: nil) }
+
+    it { is_expected.to eq [remote_rental_1] }
+  end
+
+  describe '.not_connected' do
+    subject { described_class.not_connected }
+    let(:connection_1) { create(:connection) }
+    let(:remote_rental_1) { connection_1.remote_rental }
+    let(:connection_2) { create(:canceled_connection) }
+    let(:remote_rental_2) { connection_2.remote_rental }
+    let(:remote_rental_3) { create(:remote_rental, rental: nil) }
+
+    it { is_expected.to eq [remote_rental_2, remote_rental_3] }
```

Comment:
Please add description to spec `it returns either not connected remote rentals or with canceled connection` or sth like that. 
Also better to use `match_array` as eq may cause random fails here.

Might be good idea to name objects more expressive as well - `not_connected_remote_rental` `connected_remote_rental` etc.